graph/api/src/main/java/org/jboss/windup/graph/service/ProjectService.java

```diff
@@ -67,4 +78,62 @@ public ProjectModel getOrCreateSharedLibsProject()
 
         return sharedLibsProject;
     }
+
+    public Map<ProjectModel, ProjectModel> getProjectToRootProjectMap()
```

Comment:
I think that this code needs javadoc. It isn't totally clear to me what it is doing or why it should be used vs just calling getRootProject() on a given project.